packages/cli/src/commands/auth/login-ci.js

```diff
@@ -1,33 +1,42 @@
-const os = require("os")
 const fs = require("fs")
+const os = require("os")
 const open = require("open")
+const path = require("path")
 const cli = require("cli-ux")
 const {start} = require("../../../web/index")
 const BaseCommand = require("../../utils/base-command")
+const credsPath = path.join(os.homedir(), ".rb")
 
 // login command execution
 class LoginCICommand extends BaseCommand {
   /**
    * Execute the login command
    */
-  async run() {ß
+  async run() {
     // generate random state validation
     const state = Math.floor(Math.random()*90000) + 10000;
 
     // run the local server
     start({state})
 
     // define the target url
-    const consolePath = `${BaseCommand.CONSOLE_PATH}?state=${state}&redirect_uri=http://localhost:9091/ci`
+    const consolePath = `${BaseCommand.CONSOLE_PATH}?state=${state}&redirect_uri=http://localhost:9091/`
 
     // log info to user
     this.log(`Visit this URL on this device to log in:\n\n${consolePath}\n\n`)
 
-    // open the redirect uri with the default browser
-    open(consolePath)
-
     // start spinner
     cli.ux.action.start("Waiting for authentication...")
+    // open the redirect uri with the default browser
+    open(consolePath);
+
+    process.on('exit', ( code ,signal) => {
```

Comment:
Avoid using `process.on('exit')`. process.on('exit') is designed for graceful shutdowns (for example when you hit ctrl+c on a server and you need to clear some system processes) and not standard processes that can be a part of the program

Reply:
got it, thanx